### DISCUSSION TITLE
embedding slint in android native app

### DISCUSSION THREAD

**Discussion by surya239** [slint-ui/slint#6839] [Q&A]
i am trying to place slint widget within android my android app. I created .so file for android architecture. My .so file rust methods and slit window. when i try to run mainwindow.run() or mainwindow.show() i am getting fatal signal 6 error is there any possible to show the slint widget within android app

**Comment by ogoffart** [comment]
Do you get more logs or backtrace that would tell more about this error?
I don't think anyone has done that.
I think you'll need to initialize a slint::Platform.
Similar to the platform from https://github.com/slint-ui/slint/blob/0898a5f641759ed4e9adecab1ae5b3ffde604332/internal/backends/android-activity/lib.rs#L94
